Edit python add codebytes #2821 (#2936)
Codecademy/docs: content/python/concepts/files/terms/unlink/unlink.md

# Delete file path
os.unlink("path/to/my_file.txt")
```

> ## Codebyte Example

Reviewer: same note as above regarding spacing

> In this example **delete_me.txt** is deleted using the complete file path and the `.unlink()` method:

Reviewer: ```suggestion
In this example `delete_me.txt` is deleted using the complete file path and the `.unlink()` method:
```

```codebyte/python
import os

> #Create file

Reviewer: ```suggestion
# Create file
```

f = open("/tmp/delete_me.txt", "w")
f.write("Bye bye!")
f.close()

> #List files in directory

Reviewer: ```suggestion
# List files in directory
```

> print("Files in tmp directory are: ", os.listdir("/tmp"))

Reviewer: ```suggestion
print("Files in the current directory are: ", os.listdir("/home/ccuser"))
```

Author: Hi @caupolicandiaz  because we are changing the directory here to "/home/ccuser" I just want to confirm I will also need to update line 51 as well so that delete_me.txt will be created in the correct location?

Thanks!

> #Delete entirety of the file path:

Reviewer: ```suggestion
# Delete entirety of the file path:
```

> os.unlink ("/tmp/delete_me.txt")

Reviewer: ```suggestion
os.unlink("delete_me.txt")
```

> #List files in directory after using .unlink()

Reviewer: ```suggestion
# List files in directory after using .unlink()
```

> print("Files in tmp directory are: ", os.listdir("/tmp"))

Reviewer: ```suggestion
print("Files in the current directory are: ", os.listdir("/home/ccuser"))
```
